pkg/terra/relay.go

```diff
@@ -32,9 +34,14 @@ type OCR2Spec struct {
 	IsBootstrap bool
 
 	// network data
-	NodeEndpointHTTP string
-	NodeEndpointWS   string
-	ChainID          string
+	TendermintURL      string // URL exposing tendermint RPC (default port is 26657)
+	CosmosURL          string // URL exposing cosmos endpoints (port is 1317, needs to be enabled in terra node config)
```

Comment:
are these WS urls or just http/rpc urls?

Reply:
just http/rpc

Reply:
oh interesting! they don't require an actual WS connection to "subscribe"

Reply:
did a little digging and I think it does use the WS connection if we use subscribe...it might be appending `/websocket` to the url so then it can access it for subscriptions

https://github.com/cosmos/cosmos-sdk/blob/33dbf6a795e68c744a2c312b76d41614b3563c01/client/utils.go#L79
https://pkg.go.dev/github.com/tendermint/tendermint@v0.34.15/rpc/client/http#New